it/csharp/basic/dotnet.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <RootNamespace>App</RootNamespace>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.8.2" />
```

Comment:
I believe you're missing a number of test assemblies (see the annotations on kiotamockservertests)
Also it doesn't look like this is setup as a unit test project.

Reply:
This clearly shows my lack of expertise in C# 🙂 I have been attempting to merge a couple of different `.csproj` files but I would need a link or more precise guidance to do it better.

Reply:
That said - the current setup does the expected job when invoking `dotnet test` in the folder.

Reply:
simply run drop the current csproj, run `dotnet new xunit` and then `dotnet add package <packageName>` for all the azure/kiota packages.

Reply:
Done, let me know if the new implementation looks any better.